### PR TITLE
Defined DateTimepicker constraints

### PR DESCRIPTION
**Pull request by Shriyanshagro**
*Defined [Arrival/Departure constraints for attendees #911](https://github.com/openSUSE/osem/issues/911)
 `End_date_conference> registration-arrival-date > current_date`
 `Registration-departure-date > max(start-date-conference,registration-arrival-date)`
along with prefilled dates referenced to date of conference. 

*#910 
   Constraints for Registration Period parameters  -

    end-date-conference >= registration-period-start-date >= Current_date

    registration-period-Start-date <= registration-period-End-date <= End-date-conference

Which means , admin is only allowed to set registration period in between current date and end_date of conference

*#909
Only allow creating a new Conference ,while - 
         `conference-start-day >= Current_date`
         `conference-end-day >= conference-start-day `

### CODE REVIEW

File: app/assets/javascripts/osem-datepickers.js [hunk 1]
```diff
@@ -1,33 +1,80 @@
...
-      $('#registration-arrival-datepicker').data("DateTimePicker").setMaxDate(e.date);
+  
+  $("#registration-arrival-datepicker").on("dp.change",function (e) {
+      // departure_date > start_date,arrival_date
+      if ((new Date(e.date).getTime()) > (new Date($("#registration-arrival-datepicker").attr('start_date')).getTime())){
+              console.log (e.date, "arrival")
+            $('#registration-departure-datepicker').data("DateTimePicker").setMinDate(e.date);
+      }
+      else{
+            console.log ($("#registration-arrival-datepicker").attr('start_date'),"start_date")
```
Comment (sonalkr132) [inline app/assets/javascripts/osem-datepickers.js]: Please remove `console.log`(s)

Comment (Shriyanshagro) [inline app/assets/javascripts/osem-datepickers.js]: But Why ?

Comment (sonalkr132) [inline app/assets/javascripts/osem-datepickers.js]: > But Why? 

Cause they are `console.log`, our code will work without it too.

> Does it had anything to do with warnings?

Which warnings are you talking about?

Comment (Shriyanshagro) [inline app/assets/javascripts/osem-datepickers.js]: The checks below - (Hakiri , travis)

Comment (sonalkr132) [inline app/assets/javascripts/osem-datepickers.js]: Hakiri issues are fixed on our master, so you need rebase your commits to fix that.
For travis, run the failing test locally:
```
rspec spec/features/lodgings_spec.rb
```
see which test(s) is failing and then read the test to find out why it is failing. Let us know if you need further help :)

**Comment by hennevogel** [comment]
LGTM

**Pull request merged**